sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -756,7 +756,8 @@ pi_result _pi_queue::executeOpenCommandList() {
 }
 
 static const bool FilterEventWaitList = [] {
-  const bool RetVal = std::getenv("SYCL_PI_LEVEL_ZERO_FILTER_EVENT_WAIT_LIST");
+  const char *Ret = std::getenv("SYCL_PI_LEVEL_ZERO_FILTER_EVENT_WAIT_LIST");
```

Comment:
Please document it in https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md

Reply:
Done. Thanks.